src/aria/utils/Array.js

```diff
@@ -173,6 +173,31 @@ var Aria = require("../Aria");
                     }
                 }
                 return output;
+            },
+            /**
+             * This tests whether all elements in the array pass the test given by the predicate callback function. 
+             * As long as the predicate (callback) returns truthy values, the loop will continue. However, as soon as a falsy
+             * value is returned, the function will return without analyzing the rest of the elements.
+             * @param {Array} array Array to analyze.
+             * @param {Function} Function to test each element of the array. This function receives as arguments the
+             * value, the index and the array.
+             * @param {Object} thisObject Object to use as this when executing callback.
```

Comment:
Please precise what is the default value for `thisObject` and when it is applied (e.g. `thisObject` will be set to `array` when falsy)